mdpow/workflows/dihedrals.py

```diff
@@ -418,11 +419,11 @@ def periodic_angle(df, padding=45):
 
     '''
 
-    df1 = df[df.dihedral > 180 - padding]
+    df1 = df[df.dihedral > 180 - padding].copy(deep=True)
     df1.dihedral -= 360
-    df2 = df[df.dihedral < -180 + padding]
+    df2 = df[df.dihedral < -180 + padding].copy(deep=True)
```

Comment:
I think it would be ok to leave out `deep=True` and rely on the default. I understand that you're trying to be extra cautious. Your choice.

Reply:
I would like to leave it to be cautious and explicit, but also so that someone working on this module in the future is aware that this specification could be necessary in the event that pandas makes any changes.